ext/templates/deb/postinst.erb

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
-<% if @pe -%>
-export PATH=/opt/puppet/bin:$PATH
-<% end -%>
```

Comment:
@joshcooper My understanding was that I should remove these because puppetlabs-ssl-setup already handles the path internally, and nothing else in the postinst(s) is relevant. (@kenbarber)